en/includes/guides/organization-management/inheritance-in-organizations/attribute-inheritance.md

```diff
@@ -1,59 +1,90 @@
 # User attribute inheritance
 
-Child organizations inherit all user attributes defined in the root organization. This includes both default system attributes in {{ product_name }} and any custom attributes created in the root organization.
+In {{product_name}}, child organizations inherit user attributes, user store mappings, and dialects from the root organization, ensuring consistency across the organization hierarchy.
 
-## User attributes in the local attribute dialect
+## How it works
 
-Child organizations can only update the attribute configurations that relate to secondary user stores, since each organization defines its own secondary user stores and doesn't share them with others.
+This section explains the inheritance mechanism for attributes, user store mappings, and dialects across organizations.
 
-You can find these configurations in the **Attribute Mappings** tab of an attribute.
+### User attributes
 
-### Mapped attributes
+- Child organizations inherit both the system-defined and custom attributes from the root organization.
 
-{% if product_name == "Asgardeo" %}
+- Only the root organization can create custom attributes.
 
-You can edit mappings related to user stores ( **MY USER STORE** in this example) independently for each organization.
+Organization administrators can access inherited user attributes from the {{product_name}} Console under **User Attributes & Stores** > **User Attributes**.
 
-![Attribute mappings]({{base_path}}/assets/img/guides/organization/attributes/b2b-edit-attribute-mappings.png){: width="700" style="display: block; margin: 0;"}
+### User store mappings
 
-{% else %}
+Each user store in an organization maintains mappings for user attributes. Inheritance of user store mappings work in the following way.
 
-You can edit the **PRIMARY** user store mappings shown below only in the root organization, which manages the primary user store shared across organizations.
+{% if product_name == "WSO2 Identity Server" %}
+- **Primary user store**
 
-However, you can edit mappings related to secondary user stores (**MY USER STORE** in this example) independently for each organization.
+    - Only the root organization can edit the user store mappings for the primary user store.
 
-![Attribute mappings]({{base_path}}/assets/img/guides/organization/attributes/b2b-edit-attribute-mappings.png){: width="700" style="display: block; margin: 0;"}
+    - Child organizations inherit primary user store mappings from the root organization.
 
-{% endif %}
+- **Secondary user stores**
+
+    - Child organizations can onboard their own secondary user stores.
+
+    - Child organizations have full control over user store mappings for secondary user stores, including:
+
+        - Editing mappings for user attributes inherited by the root organization.
+
+        - Whether to enable multi-valued user attributes (e.g. emailAddresses) for the secondary user stores. Only supported attributes can be enabled/disabled.
```

Comment:
```suggestion
        - Whether to enable multi-valued user attributes (e.g. emailAddresses) for the secondary user stores. This option is only available for supported attributes.
```

Shall we re-word this so that it's clear for those not familiar with these configurations?